docs/build/html/_modules/gui/_version.html

```diff
@@ -0,0 +1,624 @@
+
```

Comment:
This file should be ignored.

Reply:
@kne42 do you want to see the version somewhere else? or want to get rid of totally?